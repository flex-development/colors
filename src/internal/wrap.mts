/**
 * @file Internal - wrap
 * @module colors/internal/wrap
 */

/**
 * Create an escape sequence.
 *
 * @internal
 *
 * @this {void}
 *
 * @param {number} n
 *  Numeric code for escape sequence
 * @return {string}
 *  Escape sequence
 */
function wrap(this: void, n: number): string {
  return `\x1b[${n}m`
}

export default wrap
