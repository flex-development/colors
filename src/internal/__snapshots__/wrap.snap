// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`unit:internal/wrap > should return escape sequence 1`] = `"[0m"`;
